bindings/python/iota_sdk/types/output.py

```diff
@@ -22,6 +31,37 @@ class OutputType(IntEnum):
 
 @dataclass
 class Output():
+    """Output.
+
+    **Attributes**
```

Comment:
probably without `**` Or is it intentional?

Reply:
Have to use this hack whenever there are 10+ attributes/args unfortunately due to a limitation of `pydoc-markdown` .